### PR TITLE
ATT&CK v9 updates

### LINKED ISSUE #343: Boot or Logon Autostart Execution: Active Setup
https://attack.mitre.org/techniques/T1547/014

### LINKED ISSUE #349: Credentials from Password Stores: Windows Credential Manager
https://attack.mitre.org/techniques/T1555/004

### LINKED ISSUE #361: System Location Discovery
https://attack.mitre.org/techniques/T1614

### PR DESCRIPTION
**Pull request by mr-tz**
Collecting my updates here. Will ask for a review once I've gathered a few.

- https://attack.mitre.org/techniques/T1547/014/ - closes #343
- https://attack.mitre.org/techniques/T1555/004 - closes #349
- https://attack.mitre.org/techniques/T1614 - closes #361

### CODE REVIEW

File: host-interaction/process/create/create-process.yml [hunk 1]
```diff
@@ -23,6 +23,6 @@ rule:
       - api: ntdll.NtCreateProcess
       - api: ntdll.NtCreateProcessEx
       - api: ntdll.ZwCreateProcess
-      - api: ntdll.ZwCreateProcessEx
+      - api: ZwCreateProcessEx
```
Comment (mr-tz) [inline host-interaction/process/create/create-process.yml:26]: also present in ntoskrnl

**Comment by williballenthin** [comment]
#343

File: collection/get-geographical-location.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - Discovery::System Location Discovery [T1614]
+    examples:
+      - 9879D201DC5ACA863F357184CD1F170E:0x10001A99
+  features:
+    - or:
+      - api: GetLocaleInfo
+      - api: GetLocaleInfoEx
+      - api: GetKeyboardLayout
+      - api: GetKeyboardLayoutList
+      - api: GetKeyboardLayoutName
```
Comment (williballenthin) [inline collection/get-geographical-location.yml:18]: seems like this might be a little loose. can you convince me?

**Review by williballenthin — APPROVED** [review]

File: collection/get-geographical-location.yml (hunk 2 above)
Comment (Ana06) [reply collection/get-geographical-location.yml:18]: I think keyboard layout is not the same as location (I am using US-layout but I am not in the USA). Maybe it makes sense to split this in a different rule.

File: persistence/registry/persist-via-active-setup-registry-key.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      - Persistence::Boot or Logon Autostart Execution::Active Setup [T1547.014]
+    references:
+      - https://www.fireeye.com/blog/threat-research/2017/02/spear_phishing_techn.html
+    examples:
+      - c335a9d41185a32ad918c5389ee54235:0x4028F0
+  features:
+    - and:
+      - or:
+        - match: set registry value
+        - number: 0x80000002 = HKEY_LOCAL_MACHINE
```
Comment (Ana06) [inline persistence/registry/persist-via-active-setup-registry-key.yml:17]: Why is this needed? Doesn't `set registry value` match? 🤔

Comment (mr-tz) [reply persistence/registry/persist-via-active-setup-registry-key.yml:17]: this is a hack to cover more cases, e.g. when there's a wrapper function

File: collection/get-geographical-location.yml (hunk 2 above)
Comment (mr-tz) [reply collection/get-geographical-location.yml]: updated this rule, is this better?

File: persistence/registry/persist-via-active-setup-registry-key.yml (hunk 3 above)
Comment (Ana06) [reply persistence/registry/persist-via-active-setup-registry-key.yml:17]: would it make sense to add this to `set registry value`? 🤔

Comment (mr-tz) [reply persistence/registry/persist-via-active-setup-registry-key.yml:17]: If we add it there we cannot flexibly express setting different root keys anymore.

**Review by Ana06 — APPROVED** [review]
